apps/src/aichat/views/AichatView.tsx

```diff
@@ -8,17 +8,37 @@ import {useAppDispatch, useAppSelector} from '@cdo/apps/util/reduxHooks';
 const commonI18n = require('@cdo/locale');
 const aichatI18n = require('@cdo/aichat/locale');
 
-import {setStartingAiCustomizations} from '../redux/aichatRedux';
+import {
+  setStartingAiCustomizations,
+  clearChatMessages,
+} from '../redux/aichatRedux';
 import ChatWorkspace from './ChatWorkspace';
 import ModelCustomizationWorkspace from './ModelCustomizationWorkspace';
 import PresentationView from './presentation/PresentationView';
 import CopyButton from './CopyButton';
 import SegmentedButtons, {
   SegmentedButtonsProps,
 } from '@cdo/apps/componentLibrary/segmentedButtons/SegmentedButtons';
+import Button from '@cdo/apps/componentLibrary/button/Button';
 import moduleStyles from './aichatView.module.scss';
 import {AichatLevelProperties, ViewMode} from '@cdo/apps/aichat/types';
 
+const renderChatWorkspaceHeaderRight = (onClear: () => void) => {
+  return (
+    <div>
+      <Button
+        onClick={onClear}
+        text="Clear"
+        iconLeft={{iconName: 'paintbrush'}}
+        size="xs"
+        color="white"
+        type="secondary"
+      />
+      <CopyButton />
```

Comment:
I see that `CopyButton` is using an old version of `Button` component, meaning we have here a `<button>` inside of a `<button>`. What do you think about refactoring a `CopyButton` to use our DesignSystem new `Button` component, which will make it a little bit more clear?